py_zipkin/zipkin.py

```diff
@@ -494,10 +493,19 @@ def stop(self, _exc_type=None, _exc_value=None, _exc_traceback=None):
         # process (i.e. this zipkin_span not inside of any other local
         # zipkin_spans)
         if self.logging_context:
-            self.logging_context.stop()
-            self.logging_context = None
-            self.get_tracer().set_transport_configured(configured=False)
-            return
+            try:
+                self.logging_context.stop()
+            except Exception as ex:
+                err_msg = 'Error emitting zipkin trace, service:{}, {}'.format(
```

Comment:
nit: I wouldn't emit the service name here, it's a bit weird. We'll need to add it in yelp_pyramid anyway.